### DISCUSSION TITLE
Need help downgrading to zigbee2mqtt 1.36

### DISCUSSION THREAD

**Discussion by ryphez** [tteck/Proxmox#2863] [Help Needed]
zigbee2mqtt 1.36.1 broke several of my Jasco switches and I want to figure out how I can downgrade versions. The primary problem is in zigbee-herdsman which has a [commit ](specifically https://github.com/Koenkk/zigbee-herdsman/pull/1004)out that fixes it, but z2m hasn't been upgraded yet. I'm open to either patching forward or rolling back, but am not sure how best to do it since the script seems to checkout the latest commits on the z2m branch.

**Comment by tteck** [comment]
Or, go ahead and switch to the dev branch. That's the branch I stay on. I usually only upgrade the day of a new release, but it gives me the option to upgrade anytime things like this happens.

https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html#switch-to-the-dev-branch
